courses/machine_learning/text_classification/reusable_embeddings.ipynb

```diff
@@ -0,0 +1,972 @@
+{
```

Comment:
<p>I guess when doing these labs in the deep learning VMs we can just access Tensorboard from the Launcher?</p><p> </p><p>Other than that, this lab LGTM. Nice work!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/588/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
<p>I think so.</p>